flashfocus/compat.py

```diff
@@ -25,26 +25,27 @@ def get_display_protocol() -> DisplayProtocol:
 
 _display_protocol = get_display_protocol()
 
+# pylint: disable=unused-import
 if _display_protocol is DisplayProtocol.SWAY:
     logging.info("Detected display protocol: wayland - sway")
     from flashfocus.display_protocols.sway import (
-        disconnect_display_conn as disconnect_display_conn,
-        DisplayHandler as DisplayHandler,
-        get_focused_workspace as get_focused_workspace,
-        get_focused_window as get_focused_window,
-        list_mapped_windows as list_mapped_windows,
-        Window as Window,
+        DisplayHandler,
+        Window,
+        disconnect_display_conn,
+        get_focused_window,
+        get_focused_workspace,
+        list_mapped_windows,
     )
 elif _display_protocol is DisplayProtocol.WAYLAND:
     logging.info("Detected display protocol: wayland - other")
     raise UnsupportedWM("This window manager is not supported yet.")
 else:
     logging.info("Detected display protocol: X11")
-    from flashfocus.display_protocols.x11 import (  # noqa: F401
-        disconnect_display_conn as disconnect_display_conn,
-        DisplayHandler as DisplayHandler,
-        get_focused_workspace as get_focused_workspace,
-        get_focused_window as get_focused_window,
-        list_mapped_windows as list_mapped_windows,
-        Window as Window,
+    from flashfocus.display_protocols.x11 import (
```

Comment:
Would it be okay to keep the noqa comment here? This line causes a flake8 warning otherwise